executor/executor_test.go

```diff
@@ -455,6 +455,28 @@ func (s *testSuite) TestInsertAutoInc(c *C) {
 	r.Check(testkit.Rows(rowStr3, rowStr1, rowStr2, rowStr4, rowStr5, rowStr6))
 }
 
+func (s *testSuite) TestInsertIgnore(c *C) {
+	defer testleak.AfterTest(c)()
+	tk := testkit.NewTestKit(c, s.store)
+	tk.MustExec("use test")
+	testSQL := `drop table if exists t;
+    create table t (id int PRIMARY KEY AUTO_INCREMENT, c1 int);`
+	tk.MustExec(testSQL)
+	testSQL = `insert into t values (1, 2);`
+	tk.MustExec(testSQL)
+
+	r := tk.MustQuery("select * from t;")
+	rowStr := fmt.Sprintf("%v %v", "1", "2")
+	r.Check(testkit.Rows(rowStr))
+
+	tk.MustExec("insert ignore into t values (1, 3), (2, 3)")
```

Comment:
TiDB & MySQL have the same behavior:
mysql> create table t (id int PRIMARY KEY AUTO_INCREMENT, c1 int unique);
Query OK, 0 rows affected (0.01 sec)

mysql> insert ignore into t (c1) values (1), (2);
Query OK, 2 rows affected (0.01 sec)

mysql> select last_insert_id();
+------------------+
| last_insert_id() |
+------------------+
|                1 |
+------------------+
1 row in set (0.00 sec)

mysql> insert ignore into t values (4, 2);
Query OK, 0 rows affected (0.00 sec)

mysql> select last_insert_id();
+------------------+
| last_insert_id() |
+------------------+
|                1 |
+------------------+
1 row in set (0.00 sec)